less/c2corg_ui.less

```diff
@@ -846,6 +844,12 @@ app-simple-search {
     a.logout {
       cursor: pointer;
     }
+    @media @phone {
+      .dropdown-menu { /* right-aligned user menu on mobile */
+        left: auto;
```

Comment:
... or you could add `dropdown-menu-right` to do exactly the same thing without CSS. I did it recently for 
https://github.com/c2corg/v6_ui/blob/f349f2c0bed61d6ea09a5809e6f5cf84ef3d16ab/c2corg_ui/static/js/lang.js#L26